iree_tests/configs/models_cpu_llvm_task.json

```diff
@@ -2,20 +2,19 @@
     "config_name": "cpu_llvm_task",
     "iree_compile_flags" : [
       "--iree-hal-target-backends=llvm-cpu",
-      "--iree-llvmcpu-target-cpu-features=host"
+      "--iree-llvmcpu-target-cpu-features=host",
+      "--iree-input-demote-i64-to-i32"
     ],
```

Comment:
> Also added the --iree-input-demote-i64-to-i32 flag to the cpu models config file so that we are in line with what we run in e2eshark for any cpu code generation

I'm not sure I agree with this. I'm worried about the number of flags we have developers using, and I'd like to have more visibility into which models require which flags and why. One way we can do that is by having users of the test suite test with default flags.

In this case, the `--iree-input-` flags are sort of special in that they are generic across backends so I do see a case for using some of them... maybe in a `_compatibility_flags` test suite config, then we could diff the XFAILs between that and the `_default_flags` config.